### ISSUE TITLE
Invalid CAN frame formats

### ISSUE THREAD

**Issue by b0ch3nski** [smartgauges/qvbf#1]
`STANDART` and `EXTENDED` aren't valid CAN frame formats - should be replaced with `CAN_STANDARD` and `CAN_EXTENDED`:
```
	m_ui->cb_can_frame_format->addItem("CAN_STANDARD");
	m_ui->cb_can_frame_format->addItem("CAN_EXTENDED");
```

Also, there are more enums missing, see: https://github.com/dsch/vbftool/blob/master/vbftool/vbf.py

**Comment by smartgauges** [comment]
Hello, you are not quite right, STANDARD and EXTENDED are valid CAN frame formats for version 2.1 vbf files. For vbf files version 2.2 and above, the names CAN_STANDARD and CAN_EXTENDED are used. However, thanks for the note, I will add the names CAN_STANDARD and CAN_EXTENDED to select the CAN frame formats.

**Comment by b0ch3nski** [comment]
Thanks @smartgauges ! I stumbled upon this when edited VBF file taken from JLR SDD and after that I compared it with the original and noticed that `CAN_STANDARD` was changed to `STANDART` - that got me a little bit suspicious. But I didn't realize that the problem was related to 2.2 version of VBF format.

Nevertheless, you did an amazing work on drive2.ru - I had to use the translator all the time, but somehow managed to understand :smile: Thank you for sharing this publicly!
